hack/update-codegen.sh

```diff
@@ -43,5 +43,16 @@ done
 echo "Generating deepcopy funcs"
 ${BIN_DIR}/deepcopy-gen --input-dirs "$(codegen::join , "${FQ_APIS[@]}")" -O zz_generated.deepcopy --bounding-dirs "${APIS_PKG}" -h $BOILERPLATE
 
-echo "Generating CRDs in ./deploy/charts/trust-manager/templates"
-${BIN_DIR}/controller-gen crd schemapatch:manifests=./deploy/charts/trust-manager/templates output:dir=./deploy/charts/trust-manager/templates paths=./pkg/apis/...
+echo "Generating CRDs in ./deploy/crds"
+${BIN_DIR}/controller-gen crd schemapatch:manifests=./deploy/crds output:dir=./deploy/crds paths=./pkg/apis/...
+
```

Comment:
suggestion: I would expect that we'd have a new file in `./deploy/crds` in this PR if we're now generating CRDs there as well?